__tests__/index.test.ts

```diff
@@ -1,14 +1,14 @@
-import * as core from '@actions/core'
-import { mockClient } from "aws-sdk-client-mock";
+import * as core from '@actions/core';
 import {
-    GetSecretValueCommand, ListSecretsCommand,
```

Comment:
Could you update your tab size to 4 spaces and resubmit?

Reply:
Fixed